src/fonts/gonville_metrics.ts

```diff
@@ -100,35 +99,46 @@ export const GonvilleMetrics = {
       point: 40,
     },
     down: {
-      point: 40,
+      point: 34,
```

Comment:
Are all these small changes due to scaling / offset changes when converted to SMuFL?

What are the main visual differences?

Reply:
I am trying to make a single set of metrics so that the addition of fonts in the future is easier.